ReleaseProcessAutomation.Tests/Steps/Releases/ReleaseRcStepTests.cs

```diff
@@ -0,0 +1,231 @@
+      var nextPossibleVersions = new SemanticVersion().GetNextPossibleVersionsHotfix();
+      _continueAlphaBetaMock.Setup(_ => _.Execute(It.IsAny<SemanticVersion>(), It.IsAny<string>(), It.IsAny<string>(), false)).Verifiable();
+
+      _inputReaderMock.Setup(_ => _.ReadVersionChoice(It.IsAny<string>(), nextPossibleVersions)).Returns(new SemanticVersion());
+
+      var rcStep = new ReleaseRCStep(
+          _gitClientStub.Object,
+          _config,
+          _inputReaderMock.Object,
+          _ancestorStub.Object,
+          _msBuildInvokerMock.Object,
+          _continueAlphaBetaMock.Object,
+          _consoleStub.Object);
+
+      Assert.That(
+          () => rcStep.Execute(new SemanticVersion(), "", true, false, "hotfix/v1.3.5"),
+          Throws.Nothing);
+
+      _continueAlphaBetaMock.Verify(_ => _.Execute(It.IsAny<SemanticVersion>(), It.IsAny<string>(), It.IsAny<string>(), false), Times.Never);
+    }
+
+
+    [Test]
+    public void Execute_WithoutErrors_CallsNextStep ()
+    {
+      _gitClientStub.Setup(_ => _.IsWorkingDirectoryClean()).Returns(true);
+      _gitClientStub.Setup(_ => _.IsCommitHash("")).Returns(true);
+      _gitClientStub.Setup(_ => _.IsOnBranch("release/")).Returns(true);
+      _gitClientStub.Setup(_ => _.GetCurrentBranchName()).Returns("branchName");
+      _gitClientStub.Setup(_ => _.DoesBranchExist("prerelease/v0.0.0")).Returns(false);
+      var nextPossibleVersions = new SemanticVersion().GetNextPossibleVersionsHotfix();
+      _continueAlphaBetaMock.Setup(_ => _.Execute(It.IsAny<SemanticVersion>(), It.IsAny<string>(), It.IsAny<string>(), false)).Verifiable();
+
+      _inputReaderMock.Setup(_ => _.ReadVersionChoice(It.IsAny<string>(), nextPossibleVersions)).Returns(new SemanticVersion());
+
+      var rcStep = new ReleaseRCStep(
+          _gitClientStub.Object,
+          _config,
+          _inputReaderMock.Object,
+          _ancestorStub.Object,
+          _msBuildInvokerMock.Object,
+          _continueAlphaBetaMock.Object,
+          _consoleStub.Object);
+
+      Assert.That(
+          () => rcStep.Execute(new SemanticVersion(), "", false, false, "hotfix/v1.3.5"),
+          Throws.Nothing);
+      _continueAlphaBetaMock.Verify();
+    }
+
+    [Test]
+    public void Execute_WithoutErrors_CallsMBuildInvokeAndCommit()
+    {
+      _gitClientStub.Setup(_ => _.IsWorkingDirectoryClean()).Returns(true);
+      _gitClientStub.Setup(_ => _.IsCommitHash("")).Returns(true);
+      _gitClientStub.Setup(_ => _.IsOnBranch("release/")).Returns(true);
+      _gitClientStub.Setup(_ => _.GetCurrentBranchName()).Returns("branchName");
+      _gitClientStub.Setup(_ => _.DoesBranchExist("prerelease/v0.0.0")).Returns(false);
+      var nextPossibleVersions = new SemanticVersion().GetNextPossibleVersionsHotfix();
+      _msBuildInvokerMock.Setup(_ => _.CallMSBuildStepsAndCommit(MSBuildMode.PrepareNextVersion, new SemanticVersion())).Verifiable();
```

Comment:
Stylistic choice: if you have only void-typed calls that you wish to verify, you can move the verification to the bottom of the test, using the AAA style (arrange, act, assert). It's okay to mix these styles between tests, but you should avoid mixing them within the same test. 

Of course, if the tests vary only in their verifiable setups and some use void and others have a return statement, it might help readability of the entire fixture, to stick to verifable throughout.  => I did not check the entire fixture if that's the case, but it seems like most (all?) tests use void + verifiable.